clusterloader2/pkg/measurement/common/slos/testdata/rules.yml

```diff
@@ -0,0 +1,102 @@
+groups:
```

Comment:
Is it a copy of https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/prometheus/manifests/prometheus-rules.yaml?

If so, can we reuse that file? I think it does make sense as the rules there should be technically considered a part of measurement (it's bad that they are in some separate file, but it's a different story).
If not, can we at last add some comment to that file to keep this file in sync?

Reply:
Yes, these are the same rules as in prometheus-rules.yaml. We cannot reuse it 1:1 as prometheus-rules.yaml is actually k8s resource definition and rules are inside specification, but it should not be hard to write some logic that would extract them.